src/containers/BooksForm.js

```diff
@@ -8,21 +13,48 @@ const BooksForm = () => {
     'Learning',
     'Sci-Fi',
   ];
+  const [state, setState] = useState({ title: '', category: '' });
+  let error = false;
+
+  const handleChange = ({ target: { name, value } }) => {
+    setState({ ...state, [name]: value });
+  };
+  const { title, category } = state;
```

Comment:
- [ ] [REQUIRED] Please `move it inside handleSubmit `function under `event.preventDefault()`

Reply:
moved the line 22 `const { title, category } = state;` inside handleSubmit function